### DISCUSSION TITLE
Offline pull request reviews

### DISCUSSION THREAD

**Discussion by thomaseizinger** [community/community#47494] [Mobile] [answered]
It would be great if the app would support doing pull request reviews offline.

At the moment, making a comment requires a network connection and there is no way I can make the comments and sync them to the server later when I want to post the review.

**Comment by Yahyaezeli** [comment]
> It would be great if the app would support doing pull request reviews offline.
> 
> At the moment, making a comment requires a network connection and there is no way I can make the comments and sync them to the server later when I want to post the review.

Why are you accessing my credentials without my authorization?

**Comment by brenwebber** [comment]
👍 +1

Would be great to be able to download some PRs before a flight and review them while in the air.

**Comment by Seidahm** [comment]
Hi

**Comment by Seidahm** [comment]
Ok

**Comment by iskae** [comment, marked as answer]
Hi folks 👋 

Thank you all for reaching out! We already have an internal issue for supporting offline content on mobile 🎉 I have added this thread in it too!

Please stay tuned for our future releases!

Cheers!

**Reply by Seidahm** [reply]
> Hi folks 👋 
> 
> 
> 
> Thank you all for reaching out! We already have an internal issue for supporting offline content on mobile 🎉 I have added this thread in it too!
> 
> 
> 
> Please stay tuned for our future releases!
> 
> 
> 
> Cheers!
